src/scss/_themes.scss

```diff
@@ -23,12 +23,38 @@
 	}
 }
 
+/// Portrait card theme styles
+///
+/// @param {String} $modifier [portrait] - the modifier classname you want to use
+@mixin oCardThemePortrait($modifier: 'portrait') {
+
+	&--#{$modifier} {
+		flex-direction: column;
+		flex-wrap: nowrap;
+		justify-content: flex-start;
+	}
+
+	// Normalise the styles for an image in a portrait card
+	&--#{$modifier} &__image {
```

Comment:
It's one of the restrictions at the moment, that while the block class and modifier classes are editable, the element classes are fixed. I'm not sure how to really go about making them modifiable without some unnecessarily complicated mixins